mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
> Anyway, back to std -- maybe trimmed standard deviation, then?

Yeah, I don't know for real what works and what doesn't ... there is also [median absolute deviation](https://en.wikipedia.org/wiki/Median_absolute_deviation)